application/common/models/Password.php

```diff
@@ -225,7 +221,7 @@ public function save()
     public function validateNotPublicPassword($attribute)
     {
         /*
-         * block passwords provided in https://www.youtube.com/watch?v=3LRnjzA8auk&list=PLu5OsENIeX656zXJ96FCL169WNmvnPveo
+         * block passwords provided in https://youtu.be/WTMZYuoztoM
```

Comment:
If you want to keep the context of the YouTube playlist that video is in, you can use this URL:
https://youtu.be/WTMZYuoztoM?list=PLu5OsENIeX656zXJ96FCL169WNmvnPveo

If that doesn't matter, then the URL from @wcjr works great.

Reply:
the playlist is less likely to change, probably the better option